Makefile

```diff
@@ -142,16 +140,22 @@ help: ## Display this help
 ## Testing
 ## --------------------------------------
 
+.PHONY: test-api
+test-api: | $(GINKGO)
+test-api: ## Run API tests
+	COVERAGE_FILE="" hack/test.sh ./api
+
 .PHONY: test-nocover
 test-nocover: | $(GINKGO)
+test-nocover: | test-api
 test-nocover: ## Run tests sans coverage
-	hack/test.sh
+	hack/test.sh ./controllers ./pkg ./test ./webhooks
```

Comment:
It appears that no test cases are declared under the `./test` package. Consider removing it from this line?

Reply:
Good catch!

Reply:
Now the target tests all packages, minus `./api`, that has any test file in it directly or as a descendent.